From 553c7f61b74badf91df484450944675efd9cd485 Mon Sep 17 00:00:00 2001
From: "H.J. Lu" <hjl.tools@gmail.com>
Date: Thu, 15 Feb 2024 15:00:31 -0800
Subject: [PATCH 032/115] x86: Display -msse-check= default as none

Display -msse-check= default as none for "as --help" since its default
is none, not warning.

	PR gas/31389
	* config/tc-i386.c (md_show_usage): Change -msse-check= default
	to none.

(cherry picked from commit 7a6a03c499ad899c1d1dd93beccbb62795feb1db)
---
 gas/config/tc-i386.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gas/config/tc-i386.c b/gas/config/tc-i386.c
index 793fde25006..769a02fc691 100644
--- a/gas/config/tc-i386.c
+++ b/gas/config/tc-i386.c
@@ -16798,7 +16798,7 @@ md_show_usage (FILE *stream)
   -muse-unaligned-vector-move\n\
                           encode aligned vector move as unaligned vector move\n"));
   fprintf (stream, _("\
-  -msse-check=[none|error|warning] (default: warning)\n\
+  -msse-check=[none|error|warning] (default: none)\n\
                           check SSE instructions\n"));
   fprintf (stream, _("\
   -moperand-check=[none|error|warning] (default: warning)\n\
-- 
2.44.0

